src/render/NewGameScreen.java

```diff
@@ -16,6 +17,11 @@ public class NewGameScreen extends Screen {
     private ToggleButton yellowButton = new ToggleButton(0, COLOR_HEIGHT);
     private ToggleButton greenButton = new ToggleButton(COLOR_WIDTH, COLOR_HEIGHT);
 
+    private Company redCompany = Company.makeRed();
+    private Company blueCompany = Company.makeBlue();
+    private Company yellowCompany = Company.makeYellow();
+    private Company greenCompany = Company.makeGreen();
```

Comment:
While I guess it's possible to calculate these ahead of time and store them as fields, it's not really necessary. It'd be simpler to just call Company.make[Color]() directly from tryEnterWorld()

Reply:
Done.